libs/agno/agno/team/team.py

```diff
@@ -309,6 +313,7 @@ def __init__(
         self.session_id = session_id
         self.session_name = session_name
         self.session_state = session_state
+        self.team_session_state = team_session_state
         self.add_state_in_messages = add_state_in_messages
```

Comment:
Please where this is used also add `team_session_state` if it exists on the team. Same on the agent. 

That way the agents/team will always be "aware" of the shared team state.